src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -62,6 +65,43 @@ class ServiceTypeAhead extends Component {
     return false;
   };
 
+  renderSearchForAvailableServicePrompt = inputValue => {
+    const { isFocused } = this.state;
+    const { showError } = this.props;
+
+    if (
+      (isFocused && !inputValue && !showError) ||
+      (inputValue && inputValue.length < 2)
+    ) {
+      return (
+        <div className="dropdown" role="alert">
+          <div className="vads-u-margin--1p5">
+            {'Search for an available service'}
+          </div>
+        </div>
+      );
+    } else return null;
+  };
+
+  renderTryAnotherServicePrompt = inputValue => {
+    const { services } = this.state;
+
+    if (
+      inputValue &&
+      inputValue.length >= 2 &&
+      !services.filter(specialty => this.shouldShow(inputValue, specialty))
+        .length
+    ) {
+      return (
+        <div className="dropdown" role="alert">
+          <div className="vads-u-margin--1p5">
+            {"We couldn't find that, please try another service"}
+          </div>
+        </div>
```

Comment:
Consider refactoring this block into a function, e.g.

```js
  renderMessageDiv = message => (
       <div className="dropdown" role="alert">
          <div className="vads-u-margin--1p5">
            {message}
          </div>
        </div>
  );
```

Use in renderSearchForAvailableServicePrompt.

Could even be its own mini-component!

Reply:
I like the mini component idea!